demos/spm_face_rep/face_rep_roi_analysis.m

```diff
@@ -11,4 +11,10 @@
 opt.roi.name = {'V1v', 'V1d'};
 opt.roi.space = {'MNI', 'individual'};
 
+opt.dir.stats = fullfile(opt.dataDir, '..', 'derivatives', 'cpp_spm-stats');
+
 bidsCreateROI(opt);
+
+opt.glm.roibased.do = true;
```

Comment:
if this option is set to false, would it do the whole brain analysis with different resolution spatial preprocessing?